packages/tools/docker/cluster/docker-compose.yml

```
@@ -47,7 +47,7 @@ services:
         
     cluster0:
       container_name: cluster0
-      image: ipfs/ipfs-cluster:v0.14.5
+      image: ipfs/ipfs-cluster:v1.0.0-rc3
```

```suggestion
      image: ipfs/ipfs-cluster:v1.0.0-rc4
```

will have conflict once https://github.com/web3-storage/web3.storage/pull/1227 merged